src/Transport/ResultStatus.php

```diff
@@ -2,13 +2,13 @@
 
 declare(strict_types=1);
 
-namespace Sentry;
+namespace Sentry\Transport;
 
 /**
  * This enum represents all possible reasons an event sending operation succeeded
  * or failed.
  */
-final class ResponseStatus implements \Stringable
```

Comment:
What's the point of removing `final` here? This class represents an enum, which by definition has a fixed set of values, so it doesn't have to be extended or it would be pointless